pkg/auth/providers/azure/clients/ms_graph_client.go

```diff
@@ -302,7 +304,9 @@ func (c azureMSGraphClient) userToPrincipal(user msgraph.User) (v3.Principal, er
 
 func (c azureMSGraphClient) groupToPrincipal(group msgraph.Group) (v3.Principal, error) {
 	return v3.Principal{
-		ObjectMeta:    metav1.ObjectMeta{Name: Name + "_group://" + *group.ID},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: Name + "_group://" + *group.ID,
+		},
```

Comment:
yes! I prefer to see it this way too, for some reason the one-line version looks worse to me.

Reply:
It is more structured, the grouping is easier/better to see through the indent.